app/pods/payment-webhook/route.js

```diff
@@ -0,0 +1,28 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  api: Ember.inject.service(),
+  model() {
+    console.log("inside model hook ");
+    if(this.get('session.isAuthenticated')){
```

Comment:
Have you tried running this locally? You don't have `session` service injected into this Route, you will never enter this if statement.